web/hand_tool.js

```diff
@@ -39,9 +39,9 @@ var HandTool = (function HandToolClosure() {
 
     this.handTool = new GrabToPan({
       element: this.container,
-      onActiveChanged: function(isActive) {
-        this.eventBus.dispatch('handtoolchanged', {isActive: isActive});
-      }.bind(this)
+      onActiveChanged(isActive) {
```

Comment:
D'oh, this is a really stupid mistake on my part; thanks for spotting it!
I've updated the code to an arrow function, which is what it *should* have been from the start.